webpack.config.js

```diff
@@ -40,6 +45,26 @@ module.exports = {
       css: {
         filename: '[name].[contenthash:8].css', // CSS output filename
       },
+      // Pug preprocessor options
+      preprocessorOptions: {
+        embedFilters: {
+          // enable using markdown in Pug
+          // peer dependencies: `markdown-it`, `parse5`, `prismjs`
+          markdown: {
+            highlight: {
+              verbose: true,
```

Comment:
display in the console loaded languages when used the highlighting (`prismjs`) of code blocks